watchflower.appdata.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
FYI, ideally, you should maintain this file in your repository and ship it as the part of tarball.

Reply:
Yes everything is already in the repo:
https://github.com/emericg/WatchFlower/tree/master/assets/linux

I even build the flatpak file with a github action:
https://github.com/emericg/WatchFlower/actions/workflows/flatpak.yml

But because we need to build a tag or release, containing the flatpak files, it's always too late to fix the issue we discover on a tag or release...